File: ws/leiton/U2/WS15/ZooApp/src/main/java/ec/edu/espe/zooapp/model/Cage.java
package ec.edu.espe.zooapp.model;

import java.util.ArrayList;

/**
 *
 * @author Jose Leiton
 */
public class Cage {
    private int id;
    private ArrayList<Animal> animals;

    public Cage(int id, ArrayList<Animal> animals) {
        this.id = id;
        this.animals = animals;
    }

    @Override
    public String toString() {
        return "Cage{" + "id=" + id + ", animals=" + animals + '}';
    }

    
    
    /**
     * @return the id
     */
    public int getId() {
        return id;
    }

    /**
     * @param id the id to set
     */
    public void setId(int id) {
        this.id = id;
    }

    /**
     * @return the animals
     */
    public ArrayList<Animal> getAnimals() {
        return animals;
    }

    /**
     * @param animals the animals to set
     */
    public void setAnimals(ArrayList<Animal> animals) {
        this.animals = animals;
    }
}

File: hw/espinosa/U2/HW18 PythonInheritance/SourceCode/ec/edu/espe/zooapp/model/Cage.py
import Animal

class Cage:
    def __init__(self,id):
        self.id = id
        self.animals = []

    def add_vaccine(self,animal):
        self.animals.append(animal)
    
    def display_info(self):
        print(f"The Cage ({self.id}), has --> ")
        for animal in self.animals:
            animal.display_info()